nexus/tests/integration_tests/switch_port.rs

```diff
@@ -229,7 +227,7 @@ async fn test_port_settings_basic_crud(ctx: &ControlPlaneTestContext) {
     assert_eq!(&link0.link_name, "phy0");
     assert_eq!(link0.mtu, 4700);
 
-    let lldp0 = &roundtrip.link_lldp[0];
+    let lldp0 = link0.lldp_link_config.clone().unwrap();
```

Comment:
Is it possible to define (or derive) `PartialEq` for these structures to make the round trip testing more concise and less error prone?

Reply:
Looks like it!